undetected_chromedriver/__init__.py

```diff
@@ -129,9 +129,8 @@ def __init__(
             if user_data_dir is a path to a valid chrome profile directory, use it,
             and turn off automatic removal mechanism at exit.
 
-        browser_executable_path: str, optional, default: None - use find_chrome_executable
```

Comment:
There is still one reference to `browser_executable_path` on line 313 that you need to remove.